packages/storybook8/stories/Concepts/Spotlight/Docs.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+import CustomSpotlightCompositeText from '!!raw-loader!./snippets/CustomSpotlightComposite.snippet.tsx';
+import HideSpotlightText from '!!raw-loader!./snippets/HideSpotlight.snippet.tsx';
+import SpotlightEventText from '!!raw-loader!./snippets/SpotlightEvent.snippet.tsx';
+
+<Meta title="Concepts/Spotlight" />
+
+# Spotlight
+
+The spotlight feature enables ACS users to add spotlight to one or more participants, including themselves, in a Teams
```

Comment:
missing subheading `Overview`
<img width="615" alt="Screenshot 2024-07-29 at 4 33 58 PM" src="https://github.com/user-attachments/assets/f6cfc4d2-6ca6-4661-9dbf-588cca7fa030">

Reply:
This was intentional to have consistent formatting across the various concepts pages. Typically a short introduction doesn't have a heading, but there's a grey area between a long and short heading